lib/celluloid/group.rb

```diff
@@ -16,7 +16,7 @@ def assert_active
     def assert_inactive
       return unless active?
       if RUBY_PLATFORM == "java"
-        Celluloid.logger.warn "Group is still active"
+        Internals::Logger.warn "Group is still active"
```

Comment:
@emesepadanyi is this intentionally going around the logger? Am on phone app and cannot reference full source well atm: is `.logger` not initialized here?

Reply:
Yes. As far as I can understand, `Celluloid.logger` can be set to `nil` by the users, in case they don't want any log messages. `Internals::Logger` is sort of a wrapper to catch if `Celluloid.logger` would be nil. Anywhere else in the code `Internals::Logger` is used to trigger the logging, only this place where it is different. I think it might have been an oversight. 
That is why I think it should be like this.